tools/ci_build/github/azure-pipelines/templates/use-android-ndk.yml

```diff
@@ -0,0 +1,30 @@
+# Installs the Android NDK and sets environment variables ANDROID_NDK_HOME and ANDROID_NDK_ROOT to refer to it.
+
+  parameters:
+  - name: AndroidNdkVersion
+    type: string
+    default: "25.0.8775105"  # LTS version
```

Comment:
I don't understand: in my test change, when it failed I got an error message with "No version of NDK matched the requested version 21.0.6113669. Versions available locally: 21.4.7075529, 23.2.8568313, 24.0.8215888". So, 25.0 isn't in the list. How does it work for you?

Reply:
we install NDK 25.0 and set the relevant environment variables in this template. it's done for both Linux and Mac jobs

Reply:
If we're using a version that is not already available in the env (which I'm guessing might be the case based on Changming's list) is there any concern about network usage? The zipped windows NDK is 446MB, so a lot to download if every time the CI runs it has to do that once.

Reply:
right, it's not available in the build machine env at the moment. in the Android CI pipeline, it takes about 20 seconds to install on the Linux machines and around a minute on the Mac machines.

Reply:
As a hot fix, it looks good to me. I still hope we don't need to download the SDK every time. It's not only about network spendings, the more we download, the more likely network errors could occur.

Reply:
we can pick another version too, as long as we use the same one consistently. I just picked 25.0.8775105 because it sounds like it is supported.
https://developer.android.com/ndk/downloads

Reply:
https://github.com/actions/virtual-environments/issues/5930
looks like 25 will be added soon to the hosted agents